packages/sdk/client-services/src/packlets/spaces/notarization-plugin.ts

```diff
@@ -67,11 +88,27 @@ export class NotarizationPlugin implements CredentialProcessor {
   private readonly _processedCredentials = new ComplexSet<PublicKey>(PublicKey.hash);
   private readonly _processCredentialsTriggers = new ComplexMap<PublicKey, Trigger>(PublicKey.hash);
 
+  @logInfo
+  private readonly _spaceId: SpaceId;
+
+  private readonly _edgeClient: EdgeHttpClient | undefined;
+
+  constructor(params: NotarizationPluginParams) {
+    this._spaceId = params.spaceId;
+    if (params.edgeClient && params.edgeFeatures?.feedReplicator) {
+      this._edgeClient = params.edgeClient;
+    }
+  }
+
   get hasWriter() {
     return !!this._writer;
   }
 
-  async open() {}
+  async open() {
+    if (this._edgeClient && this._writer) {
+      this._notarizePendingEdgeCredentials(this._edgeClient, this._writer);
```

Comment:
Is it ok that this gets only called on `open`? Do we want to poll periodically in case there are any new notarization requests?

Reply:
Left an explanatory comment:
> The method is used only for adding agent feeds to spaces.
When an agent is created we can admit them into all the existing spaces. In case the operation fails
this method will fix it on the next space open.
Given how rarely this happens there's no need to poll the endpoint.